src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/self-closing-comp */
```

Comment:
- [ ] It will be great to fix the linter error instead of disabling it. Remember the linter is supposed to help you write clean code.